8. Exception handling.js

```diff
@@ -1,16 +1,15 @@
 try {
-    console.log(a);
-    let a = 3;
-  } catch (err) {
-    alert('let перед использованием нужно объявить!');
-  };
-
+  console.log(a);
+  let a = 3;
+} catch (err) {
+  alert("let перед использованием нужно объявить!");
```

Comment:
No prompts or alerts in your code. Please use direct pass value in your code and logging to console.